package/nix/generate_derivation.sh

```diff
@@ -29,9 +26,7 @@ PATCH=$(cat <<END
 
   doHaddock = false;
   jailbreak = true;
-  postInstall = ''
-    ln -s \$out/bin/elm-format-$FLAVOR \$out/bin/elm-format
-  '';
+  doCheck = false;
```

Comment:
nixpkgs doesn't provide tasty-quickcheck anymore, so turning off checks was the simplest fix.